tools/WinObjC.Packaging/WinObjC.Packageable.targets

```diff
@@ -29,13 +29,8 @@
   </PropertyGroup>
 
   <PropertyGroup>
-    <GetPackageContentsDependsOn>$(GetPackageContentsDependsOn);$(ComputeLinkInputsTargets)</GetPackageContentsDependsOn>
```

Comment:
This needs to stay to make sure that import libs are included. I need to check what this guy depends on to make sure its not trying to resolve references to get a static lib link input or updated import lib.